data/transport missions.txt

```diff
@@ -1951,3 +1951,247 @@ ship "Science Drone"
+		personality staying heroic
+		system "Kappa Centauri"
+		fleet "Large Free Worlds"
+	npc
+		government "Free Worlds"
+		personality staying heroic
+		system "Yed Prior"
+		fleet "Large Free Worlds"
+	npc
+		government "Free Worlds"
+		personality staying heroic
+		system "Pherkad"
+		fleet "Large Free Worlds"
+	on stopover
+		dialog `The last of the defenses are quickly unloaded from your ship, put into place, and activated. A pirate ship attacking the system attempts to follow you down to the surface, but the fire coming from the newly installed spaceport turrets forces the pirate to turn away and flee for orbit. Time to return to <destination>.`
+	on complete
+		payment
+		payment 5000
+		dialog `The Free Worlds captain thanks you for helping to fortify the frontier worlds, and pays you <payment>.`
+
+
+
+mission "Core Blockade"
+	minor
+	name `Bad Harvest`
+	description `The pirate world of <destination> is experiencing a famine. Warlords of the planet have threatened the Syndicate into giving them food. Deliver <cargo> to the world for <payment>.`
+	source
+		near "Almach" 2 4
+	destination "Deadman's Cove"
+	cargo "food" 287
+	to offer
+		random < 20
+		"combat rating" > 400
+	
+	on offer
+		conversation
+			`In the <origin> spaceport, you are approached by a Syndicate employee. By the looks of his uniform, he works for Syndicated Security. "Hello, Captain. You're a competent pilot, correct?" You nod your head. "Excellent. The Syndicate needs <cargo> transported to <destination>. The pirate world has seen a particularly bad harvest this past season, so much of the population is without sufficient food. Pirate warlords have threatened to begin raiding our freighters and frontier worlds if we don't provide them with food. It's cheaper to do what they want, so we've agreed to the deal, but we don't want to risk any of our own ships being destroyed. The Syndicate will pay you <payment> should you accept and complete this job."`
+			choice
+				`	"Alright, I'll get my ship ready."`
+					accept
+				`	"Sorry, but I have better things to be doing."`
+					decline
+	
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system destination
+		fleet "Large Core Pirates" 2
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system "Schedar"
+		fleet "Large Core Pirates"
+		fleet "Small Core Pirates" 3
+	
+	on complete
+		payment
+		payment 10000
+		dialog `Multiple bands of pirates, each run by a different warlord, are waiting in the spaceport when you land. They take the food from your cargo hold like animals, leaving quite the mess behind. When you return to your ship, you find that the Syndicate has deposited <payment> into your account.`
+
+
+
+mission "Pirate Blockade"
+	minor
+	name `Escape to Freedom`
+	description `Bring <bunks> escaped slaves to freedom from their pirate warlords on <destination>. The woman on <origin> promised that a person on <planet> would give you <payment>.`
```

Comment:
"Overlords" might work better than "warlords" here, since that's what they are to the slaves.

Reply:
I'll change it to oppressor instead, which I feel is far more fitting for the mission.

Reply:
Even better